function-factory-task/src/main/java/com/bobocode/FunctionFactoryTask.java

```diff
@@ -42,26 +42,33 @@ public InvalidFunctionNameException(String functionName) {
     public static void main(String[] args) {
         FunctionFactory<Integer, Integer> functionFactory = new FunctionFactory<>();
         // 1. add simple functions "square", "increment", "decrement", "negative"
+
+        functionFactory.addFunction("square", a -> a * a );
+        functionFactory.addFunction("increment", a -> a += 1);
+        functionFactory.addFunction("decrement", a -> a -+ 1);
```

Comment:
what is this?

Reply:
Hmmm, this is the Function factory task. What is the problem?)

Reply:
lol
omg
"-+"

Reply:
Fixed